src/scss/sections/_product-carousel.scss

```diff
@@ -0,0 +1,142 @@
+.product-carousel {  
+  .shop-all {
+    min-width: calc-em(154);
+    @include media-breakpoint-up(lg) {
+      min-width: calc-em(216); 
+    }
+  }
+
+  .tab-content, & {
+    overflow: hidden;
+  }
+}
+
+.product-card {
+  min-height: calc-em(533);
+  padding-top: map-get($spacers, 1);
+  @include media-breakpoint-up(lg) {
+    padding-top: map-get($spacers, 2);
```

Comment:
- why do we need paddint-top?
- we have a product-card component, so this rule should go with its code, except if only for product carousel there is an exception, but we only have 1 product carousel so please keep the code in order.
- be careful of the syntax, also media-breakpoint must be separated with 1 empty line